.travis.yml

```diff
@@ -64,10 +64,10 @@ script:
 - python setup.py --version
 
 # test new project
-- |-
-  cd /tmp/travis/testgeomapfish
-  .build/venv/bin/pcreate --scaffold=c2cgeoportal_update --ignore-conflicting-name --overwrite /tmp/travis/testgeomapfish > /dev/null  # on upgrade - don't add any argument on this command
-  cd -
+- cd /tmp/travis/testgeomapfish
+- .build/venv/bin/pcreate --scaffold=c2cgeoportal_update --ignore-conflicting-name --overwrite /tmp/travis/testgeomapfish > /dev/null  # on upgrade - don't add any argument on this command
+- make -f travis.mk checks
+- cd -
```

Comment:
What do you want to do about the errors? Should I exclude these paths from the checks?